package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "metals",
   "displayName": "Scala (Metals)",
-  "description": "Work in progress language server for Scala",
+  "description": "Language Server for Scala",
```

Comment:
I feel like we should come up with a slightly more detailed description first. Something like “Language server for Scala with rich IDE features”. I’m totally sold on removing the WIP title and updating it everywhere (website, marketplace,...)

Reply:
Isn't it a bit redundant? Other language servers follow pretty much the same pattern: "X language server", e.g. https://github.com/eclipse/eclipse.jdt.ls

Reply:
It would be nice to have “IDE” in the title to help match search queries

Reply:
We could add that to keywords

Reply:
With that in mind I'm 👍 for "Scala language server with rich IDE feature" and also adding "ide" to the keywords